src/py/flwr/driver/driver.py

```diff
@@ -57,6 +57,7 @@ def __init__(
         self.certificates = certificates
         self.channel: Optional[grpc.Channel] = None
         self.stub: Optional[DriverStub] = None
+        self.workload_id: Optional[int] = None
```

Comment:
Should we enable setting this through the constructor? Users might want to re-start the driver script after a crash, and continue to submit tasks for the same workload.

Reply:
Sound reasonable. Will do